### DISCUSSION TITLE
Lieferentenadresse

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#519] [General]
Wie pflegt ihr Lieferantenadressen aka Kreditoren/Sponsoren?

Als Nichtmitglieder könnte man sie anlegen, mit einem eigenen "Mitgliedstyp". Allerdings sind Nichtmitglieder auch nur (natürliche) Personen, wo man Vorname und Nachname angeben muss.

Ich frage auch, da das Thema E-Rechnung ja zunehmend relevanter wird und einige Unternehmen sogar unterschiedliche E-Mail-Adressen für klassische Rechnungen bzw. für XML/ZUGFeRD haben. Das möchte ich gern irgendwie pflegen.

**Comment by lenilsas** [comment]
Wenn in den Einstellungen aktiviert können auch Juristische Personen angelegt werden.

Vielleicht wäre es auch gut, wenn man für ein Mitglied mehrere Kontaktdaten angeben könnte, in Tabellenform. Bei jeder könnte man definieren wofür sie verwendet werden soll. Das würde dann auch das Problem beheben, an wen Spendenbescheinigungen und Rechnungen bei abweichendem Kontoinhaber geschickt werden sollen.

**Comment by JohannMaierhofer** [comment]
>  Das würde dann auch das Problem beheben, an wen Spendenbescheinigungen und Rechnungen bei abweichendem Kontoinhaber geschickt werden sollen.

Nicht ganz, da sich der Vollzahler ändern kann und damit auch die Mailadresse für Spendenbescheinigung, kann man die wieder nicht nehmen.
Da müsste man dann auch noch ein Gültigkeitsdatum einführen.
